packs/chef/actions/chef_install.py

```diff
@@ -20,9 +19,10 @@ class ChefInstaller(object):
         ('-d', '--download_path', {})
     ]
 
-    def install(self, install_method, options):
+    @staticmethod
+    def install(install_method, options):
         exit_code = 0
-        installer = getattr(ChefInstaller, install_method.capitalize())(options)
+        installer = getattr(sys.modules['lib'], install_method.capitalize())(options)
```

Comment:
Okay, the second way was linted successfully on my node. Waiting for 
travis. Now with all pips it's slow again(

On Tue, Jun 2, 2015 at 10:15 PM, Tomaz Muraus 
<notifications@github.com> wrote:
> In packs/chef/actions/chef_install.py:
> 
> >          exit_code = 0
> > -        installer = getattr(ChefInstaller, 
> install_method.capitalize())(options)
> > +        installer = getattr(sys.modules['lib'], 
> install_method.capitalize())(options)
> And I imagine that just changing the code to:
> 
> installer = getattr(self, install_method.capitalize())(options)
> and commenting out the import inside the class would work as well.
> 
> —
> Reply to this email directly or view it on GitHub.
>